timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/TemplateVariables.java

```diff
@@ -47,4 +53,18 @@ default TemplateVariables putLocalServerPortAmongstAllServerPorts() {
     interface TimestampPaxos {
         boolean isUseBatchPaxos();
     }
+
+    static Iterable<TemplateVariables> generateThreeNodeTimelockCluster(
+            int startingPort,
+            UnaryOperator<ImmutableTemplateVariables.Builder> customizer) {
+        List<Integer> allPorts = IntStream.range(startingPort, startingPort + 3).boxed().collect(Collectors.toList());
```

Comment:
explained offline, the `TemplateVariables` make the config that gets used in tests, so we can define the port to whatever sequence we want, and it will generate the right config. We just happen to continue from the previous tests.